ea_airflow_util/callables/sharefile.py

```diff
@@ -112,4 +113,53 @@ def sharefile_to_disk(sharefile_conn_id, sharefile_path, local_path, ds_nodash,
     if num_successes == 0:
         raise AirflowException(f"Failed transfer from ShareFile to local: no files transferred successfully!")
 
-    return date_path
\ No newline at end of file
+    return date_path
+
+
+def check_for_new_files(sharefile_conn_id: str, sharefile_path: str, expected_files: int, updated_after: datetime):
```

Comment:
We may want to rename the `expected_files` argument to `num_expected_files` or something of the like. We should also default `expected_files` and `updated_after` to falsy objects so all files are retrieved if not specified, unless this functionality goes against the scope of this feature request.

Reply:
Both good calls, thanks. Updated to `num_expected_files` and made the use of both variables optional in the callable.